importer/api/__init__.py

```diff
@@ -142,6 +144,7 @@ def import_children(course, element, parent):
         content_xml=etree.tostring(element),
         mpath=mpath,
     )
+    target = "/static/"
```

Comment:
Your approach is right, but I am wandering if this should become a module constant